source/base/MessageHandler.cpp

```diff
@@ -126,13 +126,13 @@ void MessageHandler::handleMessageReceived(Swift::Message::ref message)
             persistence_->addMessage(messageId,
                                      QString::fromStdString(fromJid),
                                      QString::fromStdString(message->getFrom().getResource()),
-                                     theBody, type, 1 );
+                                     theBody, type, isGroupMessage, 1 );
```

Comment:
try to get rid of the additional attribute in the database. See the comment at main.qml